package.json

```diff
@@ -4,19 +4,23 @@
     "dev": "wrangler dev src/index.ts",
     "deploy": "wrangler deploy --minify src/index.ts",
     "db:generate": "drizzle-kit generate",
-    "db:migrate": "drizzle-kit migrate"
+    "db:migrate": "yarn build && node dist/migrate.js",
+    "build": "tsc"
   },
   "dependencies": {
     "@fiberplane/hono": "^0.2.0",
     "@neondatabase/serverless": "^0.9.4",
```

Comment:
since we're using a general postgres driver we can remove this one? (Or then we should use this one in the migrate script)